### PR TITLE
published articles are visible in the author's page

### PR DESCRIPTION
**Pull request by vinyll**

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -39,6 +39,7 @@
 SECURITY_SEND_CONFIRMATION_TEMPLATE = 'auth/confirmation.html'
 EXPLAIN_TEMPLATE_LOADING = False
 
+DEBUG_TB_ENABLED = False
```
Comment (davidbgk) [inline settings.py]: I see why you did that and I tried (very!) hard to do that only for tests with no success 😢

File: cafebabel/tests/test_profile.py [hunk 2]
```diff
@@ -9,3 +10,20 @@ def test_confirm_user_creates_default_profile(app):
     with app.app_context():
         confirm_user(user)
     assert user.profile.name == user.email
+
+
+def test_author_profile_has_no_list_of_drafts(client, user, article):
+    login(client, user.email, 'secret')
+    response = client.get(f'/profile/{str(user.id)}/')
```
Comment (davidbgk) [inline cafebabel/tests/test_profile.py]: I don't think you need the extra `str()`

File: cafebabel/tests/test_profile.py [hunk 3]
```diff
@@ -9,3 +10,20 @@ def test_confirm_user_creates_default_profile(app):
...
+    assert response.status_code == 200
+    assert article.title not in response.get_data(as_text=True)
+
+
+def test_author_profile_has_list_of_published_articles(client, user, article):
+    login(client, user.email, 'secret')
+    article.status = 'published'
+    article.author = user
+    article.save()
+    response = client.get(f'/profile/{str(user.id)}/')
```
Comment (davidbgk) [inline cafebabel/tests/test_profile.py]: Same here.

**Review by davidbgk — CHANGES_REQUESTED** [review]

File: settings.py (hunk 1 above)
Comment (vinyll) [reply settings.py]: I'll suggest another PR with that

**Review by davidbgk — APPROVED** [review]

**Pull request merged**